test/async_sign.tests.js

```diff
@@ -41,16 +41,6 @@ describe('signing a token asynchronously', function() {
       });
     });
 
-    //Known bug: https://github.com/brianloveswords/node-jws/issues/62
```

Comment:
Did we mean to remove this test?

Reply:
This is something I changed as part of [David's comment](https://github.com/auth0/node-jsonwebtoken/pull/851#issuecomment-1314319951) and was changed when we were removing the `none` functionality all together, since we're not doing that anymore I think we can just add this back in.